packages/console/src/components/Breadcrumbs/components/BreadcrumbPopover.tsx

```diff
@@ -24,16 +24,17 @@ import { BreadcrumbEntity, BreadcrumbPopoverInterface } from '../types';
 
 const BreadcrumbPopOver = (props: BreadcrumbPopoverInterface) => {
   const history = useHistory();
+  const queryClient = useQueryClient();
 
   const {
     isLoading,
     error,
     data: popoverQueryData,
   } = useQuery(
-    `breadcrumb-list-${props.id}`,
+    `breadcrumb-list-${props.id}-${props.value}`,
```

Comment:
I don't think we should fetch the project list every time the domain changes. I think we should separate the api call and the url generation step so we don't make unnecessary network calls

Reply:
This would be a good enhancement, we'd probably need a query hash key, the formatter and data as separate params in the Breadcrumb objects. Some breadcrumbs don't change from url to url etc... 

A note here is that it only fetches once the user opens that dropdown.

Reply:
This is how the original code worked, not an enhancement. If we separate the list request from the static url generation this wouldn't be a problem